frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -311,32 +315,29 @@ const PairElement = DragSource(DRAGGABLE_TYPE.ENV_ROW, pairSource, collectSource
     return connectDropTarget(
       connectDragPreview(
         <div className={classNames('row', isDragging ? 'pairs-list__row-dragging' : 'pairs-list__row')} ref={node => this.node = node}>
+          <div className="col-md-1 col-xs-1 pairs-list__action-icon">
+            { allowSorting &&
+              <React.Fragment>{connectDragSource(<i className="pficon pficon-drag-drop pairs-list__action-icon--reorder" />)}</React.Fragment>
```

Comment:
I think the `<React.Fragment>` is unnecessary.

Reply:
I tried removing it earlier and got errors about an expression being expected when I took it out. Not sure why.